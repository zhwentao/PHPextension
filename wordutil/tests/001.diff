001+ minit
001- wordutil extension is available
002+ wordutil extension is availableRinit