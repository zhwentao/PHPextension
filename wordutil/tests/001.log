
---- EXPECTED OUTPUT
wordutil extension is available
---- ACTUAL OUTPUT
minit
wordutil extension is availableRinit
---- FAILED
