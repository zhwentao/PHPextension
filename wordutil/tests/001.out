minit
wordutil extension is availableRinit